usaspending_api/api_contracts/contracts/v2/download/awards.md

```diff
@@ -165,6 +165,8 @@ This route sends a request to the backend to begin generating a zipfile of award
 + `district` (optional, string)
 + `zip` (optional, string)
 
-### NaicsCodesObject (object)
-+ `require`: [`33`] (optional, list[string])
-+ `exclude`: [`3313`] (optional, list[string])
\ No newline at end of file
+### NAICSCodeObject (object)
++ `require`: (optional, array[string])
```

Comment:
Either the colons need to be dropped, of it can be changed to a single-line example value (which will be interpreted as a string array with one item

```markdown
+ `require` (optional, array[string])
    + `33`
```

or 

```markdown
+ `require`: `33` (optional, array[string])
```